Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -5017,6 +5018,41 @@ RefPtr<CSSStyleValue> Parser::parse_rotate_value(TokenStream<ComponentValue>& to
     return nullptr;
 }
 
+RefPtr<CSSStyleValue> Parser::parse_stroke_dasharray_value(TokenStream<ComponentValue>& tokens)
+{
+    // https://svgwg.org/svg2-draft/painting.html#StrokeDashing
+    // Value: none | <dasharray>
+    if (auto none = parse_all_as_single_keyword_value(tokens, Keyword::None))
+        return none;
+
+    // https://svgwg.org/svg2-draft/painting.html#DataTypeDasharray
+    // <dasharray> = [ [ <length-percentage> | <number> ]+ ]#
+    auto component_values = parse_a_list_of_component_values(tokens);
+    auto token_stream = TokenStream(component_values);
```

Comment:
I don't think this does anything, you should be able to use `tokens` below instead.

Reply:
😬

Done, thanks!